blog/temporal-release/index.md

```diff
@@ -0,0 +1,385 @@
+# Temporal_rs release v0.1
+
+After almost 2+ years of development, we're pleased to announce the 0.1
+release of `temporal_rs`. A calendar and time zone aware Rust date/time
+library based on ECMAScript's Temporal API.
+
+`temporal_rs` is a highly conformant implementation of the Temporal API
+in Rust that can be used in native Rust code or embedded into ECMAScript
+engines / interpreters to support their implementations.
+
+Currently, `temporal_rs` is being used by Boa, Kiesel, V8, and Yavashark
+for their Temporal implementations (but more on that later).
+
+To celebrate the 0.1 release of `temporal_rs`, we'll cover a short
+background of the Temporal implementation in Boa and why `temporal_rs`
+was split into it's own crate, we'll go over the libraries general
+design, then we'll walk through a couple brief examples of using
+`temporal_rs`, and finally we'll talk about the FFI and engine adoption.
```

Comment:
I think somewhere in the first section we want "temporal_rs is the first Rust library to be used within V8, but more on that later" and you can link to the FFI section